lib/beacon_web/components/layouts/runtime.html.heex

```diff
@@ -9,10 +9,11 @@
     <title><%= page_title(assigns) %></title>
     <%= linked_stylesheets(assigns) %>
     <%= stylesheet_tag(assigns) %>
-    <%= if !dynamic_layout?(assigns) do %>
-      <link phx-track-static rel="stylesheet" href="/assets/app.css" />
-    <% end %>
```

Comment:
@TheFirstAvenger what's the purpose of this conditional? If we're loading a user provided layout then it shouldn't load the default css provided by beacon? I'm wondering if that's still relevant because we're providing tailwind by default now.